### PR TITLE
Feature/sign up 2

### PR DESCRIPTION
**Pull request by meiravanidjar**

### CODE REVIEW

File: src/main/java/CheckersGame.java [hunk 1]
```diff
@@ -125,8 +125,10 @@ public Map<Piece, List<Move>> listPossibleMoves() {
     @Override
     public void makeMove(@NotNull User user, @NotNull Move move) {
         // Set piece to end goal
-        this.board.setPiece(move.end(), this.board.getPiece(move.start()));
-
+        if (this.board.getPiece(move.start()).getResourceName() == "King")
```
Comment (ilyash-b) [inline src/main/java/CheckersGame.java:128]: Use class check - `instanceof`

Comment (meiravanidjar) [reply src/main/java/CheckersGame.java]: fixed

**Review by ilyash-b — APPROVED** [review]
Looks good. See comments.

**Review by inna-svn — APPROVED** [review]

File: src/main/webapp/game.xhtml [hunk 2]
```diff
@@ -21,47 +24,49 @@
...
+                                                    actionListener="#{gameEndpoint.selectPiece(userSession.user, bl.piece)}">
+                                                <f:param name="id"
+                                                         value="#{gameEndpoint.idForGame(gameEndpoint.game)}"/>
+                                                <img src="Images/#{bl.piece.color.toString().toLowerCase()}Piece#{bl.piece.resourceName}.jpg"
+                                                     alt="#{bl.piece.color.toString().toLowerCase()} piece"/>
+                                            </h:commandLink>
+                                        </h:outputText>
                                     </h:outputText>
-                                    <h:outputText rendered="#{bl.getHasMoves()}">
+                                    <h:outputText rendered="#{bl.isMoveTarget}">
```
[the remark this answers is not in the document]
Comment (meiravanidjar) [reply src/main/webapp/game.xhtml:49]: I did ctrl + shift + l ,which made some spaces changes ,do it is displaying the diff like that but actually did not changed this area

**Pull request merged**